src/components/AddTodoForm.js

```diff
@@ -44,9 +58,17 @@ function AddTodoForm({ onAddTodo }) {
         {/* <button className={style.button} type="submit">
           Add
         </button> */}
+        {/* <button className={classes.button}>
+          <MdPostAdd size={18} />
+          New Post
+        </button> */}
       </form>
     </div>
   );
 }
 
+AddTodoForm.propTypes = {
+  onAddTodo: PropTypes.func,
```

Comment:
Might be nice to have prop types for all the props - I don't see todoList and setTodoList in the AddTodoForm.propTypes object